runtime/altair/src/lib.rs

```diff
@@ -1438,12 +1438,12 @@ pub struct PoolCurrency;
 impl Contains<CurrencyId> for PoolCurrency {
 	fn contains(id: &CurrencyId) -> bool {
 		match id {
-			CurrencyId::Tranche(_, _)
-			| CurrencyId::Native
-			| CurrencyId::KSM
-			| CurrencyId::ForeignAsset(_)
-			| CurrencyId::Staking(_) => false,
-			CurrencyId::AUSD => true,
+			// Deprecated, remove in the next runtime upgrade
```

Comment:
Maybe we want to specify a version number here? For me, `next` would be literal next runtime upgrade, e.g. the one which migrates `AUSD`. However, the comment wants to say `one after next` right? We can assume the migration will be part of `1028` such that it can be removed when bumping to `1029`.